src/css/custom.css

```diff
@@ -79,11 +108,13 @@ body {
   border-left: 1px solid rgba(var(--ui), 0.1);
 }
 
-a {
-  color: unset;
-  text-decoration: underline;
-}
-a:hover {
-  color: unset;
-  opacity: 0.8;
```

Comment:
why are these removed?

Reply:
I thought it would pop up more visually with those rules removed. This is what those rules do:

<img width="593" height="295" alt="image" src="https://github.com/user-attachments/assets/6f4fffe9-1f68-4b27-b8f5-9d83cfd2b271" />

With those rules removed:

<img width="590" height="306" alt="image" src="https://github.com/user-attachments/assets/f4436e6e-fbf8-41b7-b557-d30b5f661e72" />